bears/js/JSStandardBear.py

```diff
@@ -0,0 +1,25 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='standard',
+        output_format='regex',
+        output_regex=r'  (?P<filename>[^:]+):(?P<line>\d+):(?P<column>\d+):'
+                     r' (?P<message>.+)')
+class JSStandardBear:
+    """
+    JavaScript Standard Style
+
+    Find out more at <https://standardjs.com/rules.html>.
```

Comment:
This line won't be needed once we have the `SEE_MORE` attribute. See comment below.